##	\file ClosestEmptySpace.srv 
#		\brief Position of an empty spot on the table

string arm
string block_name
geometry_msgs/Pose eef_pose

---

geometry_msgs/Pose empty_pose
